app/views/conditions/dashboard.erb

```diff
@@ -0,0 +1,109 @@
+          <option value="mean_humidity">mean humidity</option>
+          <option value="mean_visibility_miles">mean visibility</option>
+          <option value="max_wind_speed_mph">max wind speed</option>
+        </select> 
+        <span>in</span>
+        <select name="increment">
+          <option value="0.5">.5</option>
+          <option value=1>1</option>
+          <option value=5>5</option>
+          <option value=10>10</option>
+        </select>
+        <span>chunks</span>
+        <input class="custom_button" type="submit" value="fetch">
+      </form>
+    </div>
+        <tr> <th> range </th> <th> average </th> <th> max </th> <th> min </th> </tr>
+        <% unless @dynamic.nil? %>
+          <% @dynamic.each do |k, v| %>
+            <tr>
+              <td> <%= k %> </td> <td> <%= v[0] %> </td> <td> <%= v[1] %> </td> <td> <%= v[2] %> </td> 
+            </tr>
+          <% end %> 
+        <% end %>
+  </table>
+</div>
+<br>
+<div>
+  <table class="table"><p class="station_title"> breakdown by high temperature in 10 degree chunks </p> 
+        <tr> <th> range </th> <th> average </th> <th> max </th> <th> min </th> </tr>
+        <% Condition.ranges_with_trips(:max_temperature_f, 10).each do |k, v| %>
+          <tr>
+            <td> <%= k %> </td> <td> <%= v[0] %> </td> <td> <%= v[1] %> </td> <td> <%= v[2] %> </td> 
+          </tr>
+        <% end %> 
+  </table>
+</div>
+<br>
+<div>
+  <table class="table"><p class="station_title"> breakdown by precipitation in half degree chunks </p> 
+        <tr> <th> range </th> <th> average </th> <th> max </th> <th> min </th> </tr>
+        <% Condition.ranges_with_trips(:precipitation_inches, 0.5).each do |k, v| %>
+          <tr>
+            <td> <%= k %> </td> <td> <%= v[0] %> </td> <td> <%= v[1] %> </td> <td> <%= v[2] %> </td> 
+          </tr>
+        <% end %> 
+  </table>
+</div>
+<br>
+<div>
+  <table class="table"><p class="station_title"> breakdown by mean wind speed in 4 mph chunks </p> 
+        <tr> <th> range </th> <th> average </th> <th> max </th> <th> min </th> </tr>
+        <% Condition.ranges_with_trips(:mean_wind_speed_mph, 4).each do |k, v| %>
+          <tr>
+            <td> <%= k %> </td> <td> <%= v[0] %> </td> <td> <%= v[1] %> </td> <td> <%= v[2] %> </td> 
+          </tr>
+        <% end %> 
+  </table>
+</div>
+<br>
+<div>
+  <table class="table"><p class="station_title"> breakdown by mean visibility in 4 mph chunks </p> 
+        <tr> <th> range </th> <th> average </th> <th> max </th> <th> min </th> </tr>
+        <% Condition.ranges_with_trips(:mean_visibility_miles, 4).each do |k, v| %>
+          <tr>
+            <td> <%= k %> </td> <td> <%= v[0] %> </td> <td> <%= v[1] %> </td> <td> <%= v[2] %> </td> 
+          </tr>
+        <% end %> 
+  </table>
+</div>
+<div>
+  <h3><p class="station_title"> weather on the day with the highest rides </p> 
+    <% condition_most = Trip.condition_on_day_with_most_rides.first %></h3>
```

Comment:
again, our view's responsibility is to display data, not to set variables or call our models